src/commands/init.ts

```diff
@@ -70,7 +70,8 @@ export default class Init extends Command {
       const files = readTemplates({
         from: [path.resolve(__dirname, '..', 'templates', 'init')],
         to: flags.projectDir,
-        exclude: initType === InitType.existingProject ? ['test'] : []
+        exclude:
+          initType === InitType.existingProject ? ['test', 'jest.config.js', 'jest.integration-test.config.js', 'jets.unit-test.config.js'] : []
```

Comment:
👍

It might be worth it to move line 74 out of the function call:
```ts
const exclude = initType === InitType.existingProject ? ['test', 'jest.config.js', 'jest.integration-test.config.js', 'jets.unit-test.config.js'] : [];
const files = readTemplates({ 
  ...
  exclude,
  ...
});
```